### DISCUSSION TITLE
Image Url scraping is not consistent

### DISCUSSION THREAD

**Discussion by gary9872** [kevinzg/facebook-scraper#1049] [Q&A]
When scraping a post that has [image] data, sometimes it return an improper url however other times it returns a correct one. Is there a way to solve this?

Thanks

Correct one starts with

https://scontent-yyz1-1.xx.fbcdn.net/v/t39.30808-6/365124598_10087243624626795_7698170478862146881_n.jpg?stp=cp0_dst-jpg_e15 .....

Incorrect one starts with:

https://m.facebook.com/photo/view_full_size/fbid=10087243661293458&ref_component=mbasic_photo_permalink&ref_page=%2Fwap%2Fphoto.php&refid=13&_ft_=encrypted_tracking_data.0AY9ksb9Jb1TgfYtoHhyVZXzDuboYi2Yq1oJAM6Rj ....

**Comment by Charlotte-br560** [comment]
Certainly! When encountering inconsistent image URLs during web scraping, it might be beneficial to check how similar challenges are addressed in projects like Crawlbase. Exploring their documentation or community discussions could provide insights into handling variations in image URLs and implementing solutions to ensure consistent and proper retrieval. Good luck resolving this issue!
